markdown-mode.el

```diff
@@ -8502,50 +8528,6 @@ newline after."
         (setq new-to (point)))
     (cl-values new-from new-to)))
 
-(defun markdown-check-change-for-wiki-link (from to)
```

Comment:
Please don't remove public interfaces, this causes breaking backward compatibility